### PR TITLE
Docs: setup rearrangements.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: setup rearrangements.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/getting-started/setup/configuration.md [hunk 1]
```diff
@@ -85,18 +80,14 @@ ClassicEditor.create( document.querySelector( '#editor' ), {
...
-CKEditor&nbsp;5 currently supports more than 40 languages, while many more community translations are also available. This also includes RTL and CJK languages. The translation packs are available both as npm packages and from CDN. Refer to the {@link getting-started/setup/ui-language UI language} guide for details.
-
+<!-- If this section to be removed - it is linked in the following places:
+features/remove-format.html
+41.3.1/framework/architecture/plugins.html
+getting-started/installation/react/react.html
+getting-started/legacy/installation-methods/quick-start.html
+getting-started/legacy/legacy-integrations/react.html
+getting-started/setup/toolbar.html:
+-->
```
Comment (Witoso) [inline docs/getting-started/setup/configuration.md:90]: Not sure what this means...

File: docs/getting-started/setup/configuration.md [hunk 2]
```diff
@@ -1,14 +1,9 @@
 ---
-# Scope:
-# * Introduction to setting configurations.
-# * Introduction to the top and must-know configurations.
-# * Point where to find the list of configuration options.
-
 category: setup
-menu-title: Configuration
+menu-title: Configuring features
 meta-title: Configuration | CKEditor 5 documentation
```
Comment (Witoso) [inline docs/getting-started/setup/configuration.md:4]: Needs change.

File: docs/getting-started/setup/configuration.md [hunk 3]
```diff
@@ -1,14 +1,9 @@
...
 meta-description: Learn how to configure CKEditor 5. 
-order: 10
+order: 20
 modified_at: 2024-05-06
 ---
```
Comment (Witoso) [inline docs/getting-started/setup/configuration.md:9]: H1 needs to change.

Comment (Witoso) [reply docs/getting-started/setup/configuration.md:9]: I wouldn't move people to framework in the paragraph of adding features. TBH, I would remove all links to framework, and remove the word build. Add a link to the features section where you could find all the features.

> Note that some features may require more than one plugin to run. You will find the plugin-per-feature listing in the {@link framework/architecture/plugins#plugins-and-html-output Plugins and HTML output} guide.

This also needs rewording and not linking to HTML output.

**Review by Witoso — CHANGES_REQUESTED** [review]

File: docs/getting-started/setup/configuration.md (hunk 1 above)
Comment (godai78) [reply docs/getting-started/setup/configuration.md:90]: It's a notice for future works.

File: docs/getting-started/setup/configuration.md (hunk 3 above)
Comment (godai78) [reply docs/getting-started/setup/configuration.md:9]: In commit.

File: docs/getting-started/setup/configuration.md (hunk 2 above)
Comment (godai78) [reply docs/getting-started/setup/configuration.md]: In commit.

**Pull request merged**